docs_user/main.adoc

```diff
@@ -22,12 +22,6 @@ include::assemblies/assembly_adopting-openstack-control-plane-services.adoc[leve
 
 include::assemblies/assembly_adopting-the-data-plane.adoc[leveloffset=+1]
 
-include::assemblies/assembly_migrating-ceph-rbd.adoc[leveloffset=+1]
-
-include::assemblies/assembly_migrating-ceph-rgw.adoc[leveloffset=+1]
-
-include::modules/proc_migrating-ceph-mds.adoc[leveloffset=+1]
-
-include::assemblies/assembly_migrating-ceph-monitoring-stack.adoc[leveloffset=+1]
-
 include::assemblies/assembly_migrating-the-object-storage-service.adoc[leveloffset=+1]
+
+include::assemblies/assembly_migrating-ceph-cluster.adoc[leveloffset=+1]
```

Comment:
@klgill so the idea here is to have an entrypoint for the main ceph doc, and from there we can include all the sections related to the components:

```
main osp doc
 -> include ceph
     -> include cardinality
     -> include monitoring
     -> include mds
     -> include rgw
     -> include mgr
     -> include mon
```  
Each of them should be an `assembly` where we can add an intro and include a `module` with the associated procedure.
If we follow this kind of pattern it results clean and easy to maintain.

Reply:
We can plan and discuss about more updates in a follow up, but the main idea is to have something easy to follow from customers point of view.